experimental/dds/tree2/src/test/feature-libraries/sequence-field/compose.spec.ts

```diff
@@ -50,9 +51,34 @@ describe("SequenceField - Compose", () => {
 						title.startsWith("((delete, insert), revive)") ||
 						title.startsWith("((move, insert), revive)") ||
 						!SF.areComposable([taggedA, taggedB, taggedC])
+					) {
+						// These changes do not form a valid sequence of composable changes
```

Comment:
This comment seems inconsistent with the more detailed one below.

Reply:
I think I need help seeing how. I admittedly don't recall what's wrong with `((delete, insert), revive)` and `((move, insert), revive)`, but the fact that they were grouped with `!SF.areComposable([...])` makes me think they similarly didn't make sense together.